compiler/src/compile.rs

```diff
@@ -984,6 +984,64 @@ impl<O: OutputStream> Compiler<O> {
         Ok(())
     }
 
+    fn option_stmt_to_bool(&mut self, stmts: &Option<ast::Suite>) -> bool {
+        match &stmts {
+            Some(stmts) => self.find_ann(stmts),
+            None => false,
+        }
+    }
+
+    fn find_ann(&mut self, body: &[ast::Statement]) -> bool {
```

Comment:
I don't think this needs to be `&mut self`:
```suggestion
    fn find_ann(&self, body: &[ast::Statement]) -> bool {
```

Reply:
Done, Thank you